### PR TITLE
pagination implementation

### PR DESCRIPTION
**Pull request by TizianoBruno96**
created pagination with skip and take

### CODE REVIEW

File: Controllers/HomeController.cs [hunk 1]
```diff
@@ -16,11 +16,15 @@ public HomeController(ILogger<HomeController> logger, BookRepository bookReposit
         _logger = logger;
     }
 
-    public async Task<IActionResult> Index()
+    public async Task<IActionResult> Index(int skip = 0, int take = 5)
     {
-        List<Book> books = await _bookRepository.GetAll();
-        ViewBag.Books = books;
+        List<Book> books = await _bookRepository.GetAll(skip, take + 1);
```
Comment (Raevaen) [inline Controllers/HomeController.cs:21]: Why take + 1?

File: Controllers/HomeController.cs [hunk 2]
```diff
@@ -16,11 +16,15 @@ public HomeController(ILogger<HomeController> logger, BookRepository bookReposit
...
     {
-        List<Book> books = await _bookRepository.GetAll();
-        ViewBag.Books = books;
+        List<Book> books = await _bookRepository.GetAll(skip, take + 1);
+        
+        ViewBag.Books = books.Take(take).ToList();
+        ViewBag.skip = skip;
+        ViewBag.take = take;
         
+        ViewBag.hasNextPage = books.Count > take;
```
Comment (Raevaen) [inline Controllers/HomeController.cs:27]: View
ViewBag properties with capital letters

**Review by Raevaen — CHANGES_REQUESTED** [review]

File: Controllers/HomeController.cs (hunk 1 above)
Comment (TizianoBruno96) [reply Controllers/HomeController.cs]: to make it work

File: Controllers/HomeController.cs (hunk 2 above)
Comment (TizianoBruno96) [reply Controllers/HomeController.cs]: fixed

**Pull request merged**